dashboard/app/main.go

```diff
@@ -1229,48 +1237,70 @@ func makeUIJob(job *Job, jobKey *db.Key, bug *Bug, crash *Crash, build *Build) *
 	return ui
 }
 
-func fetchErrorLogs(c context.Context) ([]byte, error) {
+func fetchErrorLogs() ([]byte, error) {
+	if !appengine.IsAppEngine() {
+		return nil, nil
+	}
+
 	const (
-		minLogLevel  = 3
-		maxLines     = 100
-		maxLineLen   = 1000
-		reportPeriod = 7 * 24 * time.Hour
+		maxLines   = 100
+		maxLineLen = 1000
 	)
-	q := &log.Query{
-		StartTime:     time.Now().Add(-reportPeriod),
-		AppLogs:       true,
-		ApplyMinLevel: true,
-		MinLevel:      minLogLevel,
+
+	ctx := context.Background()
+	projId := os.Getenv("GOOGLE_CLOUD_PROJECT")
+
+	adminClient, err := logadmin.NewClient(ctx, projId)
+	if err != nil {
+		return nil, errors.New("failed to create the logging client")
 	}
-	result := q.Run(c)
-	var lines []string
-	for i := 0; i < maxLines; i++ {
-		rec, err := result.Next()
-		if rec == nil {
+	defer adminClient.Close()
+
+	const name = "appengine.googleapis.com%2Frequest_log"
+
+	// default filter fetch last 24h
```

Comment:
I think we want at least a week. We don't even look at the admin page every day.

Reply:
Done.